scipy/stats/tests/test_continuous_basic.py

```diff
@@ -55,7 +55,8 @@
                 'kappa4', 'powerlognorm', 'tukeylambda'}
 xslow_fit_mle = {'gausshyper', 'ncf', 'ncx2', 'recipinvgauss', 'vonmises_line'}
 xfail_fit_mle = {'ksone', 'kstwo', 'trapezoid', 'truncpareto', 'irwinhall'}
-skip_fit_mle = {'levy_stable', 'studentized_range'}  # far too slow (>10min)
+skip_fit_mle = {'dpareto_lognorm',  # temporary skip
```

Comment:
MLE actually passes here. The fits are not bad. Although the parameter values may be quite different than the originals, the value of the NLLF is lower, and that's all we can claim to do.

Reply:
Multimodal likelihood? If NLLF is lower, let's not skip then I would say. Or does the test compare against the original parameters?

Reply:
I see the test failures in CI now. Lets gi with the skips.

Reply:
> Multimodal likelihood?

Not necessarily. Sometimes it's just really flat.

> I see the test failures in CI now

Actually they are passing; it's just fail_slow. But yeah, I'll move this to `xslow`.